README.md

```diff
@@ -8,39 +8,15 @@ To learn & comprehend **Agile Software Development Methodology** via the simulat
 
 For your trophy case you will be able to add: **A fork of this repository** (if completed in total) to showcase your studies in Agile alongside with **a completed web application repository of your own creation**. 
 
-
```

Comment:
Remove this file from your push and pushes in the future or add it to your gitignore. That way, the main version can stay in main and your personal one can stay with you